homeassistant/bootstrap.py

```diff
@@ -336,22 +423,56 @@ def from_config_file(config_path: str,
     hass.config.config_dir = config_dir
```

Comment:
That will not work since we need the hass.loop for create a executor for create hass.loop 😄